libs/utils/android/benchmark.py

```diff
@@ -203,9 +211,15 @@ def __init__(self):
             raise
 
         self._log.info('=== Execution...')
-        self.wl.run(out_dir=self.out_dir,
-                    collect=self._getBmCollect(),
-                    **self.bm_params)
+        for iter in range(self.bm_iterations):
```

Comment:
It looks good, but I wonder how should we handle it if people would want at some point to reboot between iterations. At the moment we have the possibility to reboot between test run (with different governors and tests within the benchmark) but I wonder if we should use the reboot variable to control a potential reboot between iterations too. What do you think?